week2/homework/src/index.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+'use strict';
+// TODO: Write the homework code in this file
+let fs = require('fs');
+let command = process.argv[2];
+let value = process.argv[3];
+let index = parseInt(value);
+let updateTask = process.argv[4];
+
+const functionality = require('./functionality');
+
+switch (command) {
+  case 'list':
+    console.log(functionality.todoList());
+    break;
+  case 'add':
+    functionality.add(value);
```

Comment:
Because you have removed the `console.log`s from the functionality now, you are not giving any response to the user.